lib/elixir_sense/providers/suggestion/complete.ex

```diff
@@ -473,7 +473,7 @@ defmodule ElixirSense.Providers.Suggestion.Complete do
     for {{k, nil, nil}, _} <- env.mods_and_funs, do: Atom.to_string(k)
   end
 
-  defp match_module_funs(mod, hint, include_builtin, env) do
```

Comment:
Why are those funs needed public? They weren't intended as an API

Reply:
Ah, yes so I ended up "needing" them in the ash extension I was writing, but I'm happy to mark these as private and then either
a.) work w/o them 
b.) make a separate PR to expose them as public